src/ray/core_worker/experimental_mutable_object_manager.cc

```diff
@@ -290,12 +294,14 @@ Status MutableObjectManager::ReadAcquire(const ObjectID &object_id,
   size_t total_size = channel->mutable_object->header->data_size +
                       channel->mutable_object->header->metadata_size;
   RAY_CHECK_LE(static_cast<int64_t>(total_size), channel->mutable_object->allocated_size);
+  std::shared_ptr<ChannelBuffer> channel_buffer = std::make_shared<ChannelBuffer>(
```

Comment:
Perhaps we should name it something like MutableBuffer to distinguish from the normal PlasmaBuffers.

Reply:
How about `MutableObjectBuffer`? `MutableBuffer` sounds like the buffer is mutable, which is a bit ambiguous.

Reply:
Yes, that sounds good!